documentation/index.html

```diff
@@ -33,6 +33,12 @@ <h5>HOME</h5>
             <a class="navbar navbar-m-lr-1" href="#">
                 <i class="fa-brands fa-twitter"></i>
             </a>
+            <a id="dark-mode" class="navbar navbar-m-lr-1 m-r-2rem" href="#">
```

Comment:
I don't understand the class names here, e.g., `m-r-2rem`. It can be a bit more descriptive.